src/pages/billing/Project.js

```diff
@@ -258,32 +335,27 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
     setProjectUsers(projectUsers)
   })
 
-
   // Lifecycle
-  Utils.useOnMount(() => {
-    refresh()
-    loadBillingInfo()
-  })
+  Utils.useOnMount(refresh)
 
-  useEffect(() => {
-    StateHistory.update({ projectUsers })
-  }, [projectUsers])
+  useEffect(() => StateHistory.update({ projectUsers }), [projectUsers])
 
+  // There's some madness going on when using state variables in polling effects - the reference
+  // never updates (i.e. it's bound to the value that the component was first mounted with).
+  // `useGetter` works around this and I have no idea why.
```

Comment:
It's because the function is called in a `while` loop, so it's all technically one execution, bound with the original value.